sys/psa_crypto/doc.txt

```diff
@@ -339,12 +339,37 @@
  * ### MAC
  * - Base: psa_mac
  *
+ * #### HMAC MD5
+ * - psa_mac_hmac_md5
+ * - psa_mac_hmac_md5_custom_backend
+ * - psa_mac_hmac_md5_backend_riot
+ *
+ * #### HMAC SHA 1
+ * - psa_mac_hmac_sha_1
+ * - psa_mac_hmac_sha_1_custom_backend
+ * - psa_mac_hmac_sha_1_backend_riot
+ *
+ * #### HMAC SHA 224
+ * - psa_mac_hmac_sha_224
+ * - psa_mac_hmac_sha_224_custom_backend
+ * - psa_mac_hmac_sha_224_backend_riot
+ *
  * #### HMAC SHA 256
  * - psa_mac_hmac_sha_256
  * - psa_mac_hmac_sha_256_backend_periph
```

Comment:
Why? Cryptocell has support for this.

Reply:
I plan to implement proper support for hardware acceleration in a follow-up PR. I will update the PR description so that this PR will not be merged until then.